ext/opcache/jit/zend_jit_helpers.c

```diff
@@ -27,9 +27,14 @@ static ZEND_COLD void undef_result_after_exception(void) {
 	}
 }
 
-static ZEND_COLD void zend_jit_illegal_offset(void)
+static ZEND_COLD void zend_jit_illegal_array_offset(zval *offset)
```

Comment:
this offset pointer can become const too: `const zval *offset`.

Reply:
Thanks for the correction!